### DISCUSSION TITLE
How to integrate with existing app?

### DISCUSSION THREAD

**Discussion by kokotom** [RobinTail/express-zod-api#832] [Q&A] [answered]
I want to mount express-zod-api as part of an existing express app?
How can I do it?

**Comment by RobinTail** [comment, marked as answer]
Hello @kokotom ,

Here is the approach on connecting `express-zod-api` to your own `express` app:
https://github.com/RobinTail/express-zod-api#connect-to-your-own-express-app

consider this as a suggestion for complete migration strategy:
https://github.com/RobinTail/express-zod-api/discussions/647#discussioncomment-3891254

I hope it will work for you

**Reply by kokotom** [reply]
Hey! I partially figured it out.
When I removed the notFoundHandler It worked.
It probably overrode all other routes.

**Reply by RobinTail** [reply]
ah, yes, that one should be attached in the very end.
Otherwise it will override.
